packages/cli/cloud/src/services/cli-api.ts

```diff
@@ -111,8 +114,22 @@ export async function cloudApiFactory(token?: string): Promise<CloudApiService>
       return axiosCloudAPI.get('/user');
     },
 
-    config(): Promise<AxiosResponse<CloudCliConfig>> {
-      return axiosCloudAPI.get('/config');
+    async config(): Promise<AxiosResponse<CloudCliConfig>> {
+      try {
+        const response = await axiosCloudAPI.get('/config');
+
+        if (response.status !== 200) {
+          throw new Error('Error fetching cloud CLI config from the server.');
+        }
+
+        return response;
+      } catch (error) {
+        logger.error(
+          "🥲 Oops! Couldn't retrieve the cloud CLI config from the server. Please try again."
+        );
+
+        throw error;
```

Comment:
wouldn't this log two errors in the user's terminal if the action is try..catching this ?

Reply:
Yes, it could output something like:

- "🥲 Oops! Couldn't retrieve the cloud CLI config from the server. Please try again." 
- "Login action failed."

Reply:
Could we only have one log for UX ?

Reply:
Sure, if it's too much details for the `error` log level (or if we have a "one error line max per command" rule), I'll output it to `debug` instead